package.json

```diff
@@ -271,6 +271,7 @@
           "default": {
             "amd64": [],
             "arm32v7": [],
+            "arm64v8":[],
```

Comment:
What will happen if the module doesn't have arm64v8 dockerfile?

Reply:
just like windows-amd64, some module templates doesn't contain  windows-amd64 dockerfile, if user change platform to windows-amd64, the deployment.template would show error tips